app/views/costumes/_info_window.html.erb

```diff
@@ -1,4 +1,9 @@
-<h2><%= costume.name %></h2>
+<div>
+  <% if costume.photo.attached? %>
+    <%= cl_image_tag costume.photo.key, width: 80, height: 80 %>
+  <% else %>
+    <%= [image_tag("https://res.cloudinary.com/le-wagon-london-campus-batch-904/image/upload/v1660254929/justin-jason-NZlbve4F1xk-unsplash_exnsiv.jpg"), image_tag("https://res.cloudinary.com/le-wagon-london-campus-batch-904/image/upload/v1660254926/karsten-winegeart-ewfHXBcuFA0-unsplash_cffclz.jpg"), image_tag("https://res.cloudinary.com/le-wagon-london-campus-batch-904/image/upload/v1660254926/kerwin-elias-7-ToFEHzMNw-unsplash_ffseqw.jpg"), image_tag("https://res.cloudinary.com/le-wagon-london-campus-batch-904/image/upload/v1660254926/karsten-winegeart--_gMskl-uoc-unsplash_ldomqk.jpg")].sample %>
```

Comment:
FYI for next time - put things like this in the controller. There shouldn't really be logic like this in the front end. For now (and due to deadlines) this is good to use.